docs/book/v5/reference/account-anonymization.md

```diff
@@ -1,9 +1,13 @@
 # Account anonymization
 
+## Premise
+
 According to the GDPR, companies that record personal data from EU citizens must delete said data if its owner requests its deletion.
 An alternative is to anonymize the data, according to [this article](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/dealing-citizens/do-we-always-have-delete-personal-data-if-person-asks_en).
 
-## What is 'personal data'?
+## Definition
+
+### What is 'personal data'?
```

Comment:
What is Personally identifiable information (PII)  ?

Reply:
Personally identifiable information (PII) is any information connected to a specific individual that can be used to uncover that individual's identity, such as their social security number, full name, email address or phone number.